client/my-sites/earn/index.ts

```diff
@@ -6,7 +6,7 @@ import earnController from './controller';
 export default function () {
 	page( '/earn', siteSelection, sites, makeLayout, clientRender );
 	page( '/earn/stats', siteSelection, sites, makeLayout, clientRender );
-	page( '/earn/customers', siteSelection, sites, makeLayout, clientRender );
+	page( '/earn/supporters', siteSelection, sites, makeLayout, clientRender );
```

Comment:
You can leave a redirection behind with `page( '/earn/customers', '/earn/supporters' );`

Reply:
If this page had been around a while, I think we'd want to do this. But it was just added last week, and this is the only reference / nav to the page. So I think it should be fine to just change it.

Reply:
Yeah, sounds fine! 👍